build-system/tasks/performance/stats.js

```diff
@@ -76,7 +76,8 @@ const q1 = (array) => {
 const q3 = (array) => {
   sort(array);
   const mid = Math.ceil(array.length / 2);
-  const half = array.slice(mid);
+  // account for arrays of length 1
+  const half = mid < array.length ? array.slice(mid) : array;
```

Comment:
For your consideration, another version that might work:
```
const mid = Math.floor(array.length / 2);
array.splice(0, mid);
return median(array);
```

Since this version allocates the middle element in arrays with odd numbered sizes to the upper half, you'd have to use `Math.floor` in `q1()` as well.